src/zksync/bridge-utils/approveL1Erc20Token.ts

```diff
@@ -0,0 +1,55 @@
+import type { Address } from 'abitype'
+import { erc20Abi } from 'abitype/abis'
+import {
+  type WriteContractParameters,
+  writeContract,
+} from '../../actions/index.js'
+import type { Client } from '../../clients/createClient.js'
+import type { Transport } from '../../clients/transports/createTransport.js'
+import type { Account, GetAccountParameter } from '../../types/account.js'
+import type { Chain, GetChainParameter } from '../../types/chain.js'
+import type { Hash } from '../../types/misc.js'
+import { parseAccount } from '../../utils/index.js'
+import type { Overrides } from '../types/deposit.js'
+
+export type ApproveErc20L1Parameters<
+  TChain extends Chain | undefined = Chain | undefined,
+  TAccount extends Account | undefined = Account | undefined,
```

Comment:
```suggestion
  chain extends Chain | undefined = Chain | undefined,
  account extends Account | undefined = Account | undefined,
```

Generics should be lowercased. Can you apply this convention elsewhere too?